docs/rackhd/event_notification.rst

```diff
@@ -299,4 +303,50 @@ An example of multiple filters
 Web Hook APIs
 ^^^^^^^^^^^^^^^
 
-**TODO** (The APIs of web hooks developments are ongoing, this section wiil be updated when they are finished).
+
+**Create a new hook**
+
+
+.. code-block:: REST
+
+    POST /api/2.0/hooks
+    {
+        "url": "http://www.abc.com/def"
```

Comment:
POST error code should be a common design, I am hesitated to add more description here since we already highlight the uniqueness of url.